tests/components/MapboxMap.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { ComponentMeta, Story } from '@storybook/react';
+import _ from 'lodash';
```

Comment:
nit but I think I prefer importing directly from `lodash/clonedeep`? it doesn't really matter here but I think(?) the bundle size is a little smaller that way